src/core/components/choice-card/stories/single-state-controlled-example.tsx

```diff
@@ -53,6 +53,60 @@ export const singleStateControlled = () => {
 	)
 }
 
+export const singleStateControlledWithColumns = () => {
+	const [selected, setSelected] = useState<string | null>("green")
+
+	return (
+		<div css={medium}>
+			<div css={spaced}>
+				<ChoiceCardGroup name="colours" columns={3}>
+					<ChoiceCard
+						value="red"
+						label="Red"
+						id="default-red"
+						checked={selected === "red"}
+						onChange={() => setSelected("red")}
+					/>
+					<ChoiceCard
+						value="green"
+						label="Green"
+						id="default-green"
+						checked={selected === "green"}
+						onChange={() => setSelected("green")}
+					/>
+					<ChoiceCard
+						value="blue"
+						label="Blue"
+						id="default-blue"
+						checked={selected === "blue"}
+						onChange={() => setSelected("blue")}
+					/>
+					<ChoiceCard
+						value="orange"
+						label="Orange"
+						id="default-orange"
+						checked={selected === "orange"}
+						onChange={() => setSelected("orange")}
+					/>
+					<ChoiceCard
+						value="yellow"
+						label="Yellow"
+						id="default-yellow"
+						checked={selected === "yellow"}
+						onChange={() => setSelected("yellow")}
+					/>
+				</ChoiceCardGroup>
+			</div>
+			<span css={message}>{selected} is selected</span>
+		</div>
+	)
+}
+
+
```

Comment:
Please could we move this into its own module, perhaps called `with-columns.tsx`?